docs/examples.rst

```diff
@@ -35,3 +35,10 @@ DynamoDB
 
    examples/dynamo_db/insertions
    examples/dynamo_db/tables
+
+Cognito
```

Comment:
Technically, your examples cover parts of the Amazon Cognito Identity Provider (`cognito-idp`) API, which is part of the larger family of Cognito APIs. You might want to make that explicit, e.g. by choosing the heading `Cognito Identity Provider` and using the term `cognito_idp` in directory and file names.